submodules/strategyHolidays/strategyHolidays.js

```diff
@@ -456,6 +576,56 @@ define(function(require) {
 			return holidaysData;
 		},
 
+		/**
+		 * Returns an array of arrays containing intervals for each day of the week.
+		 * @param  {Array} csvData
+		 * @return {Array[]}
+		 * Intervals are extracted from CSV data and are not necessarily linear and/or concurrent.
+		 */
+
+		strategyHolidaysExtractDatesFromCsvData: function(csvData) {
+			var self = this,
+				months = _.map(self.i18n.active().strategy.holidays.months, function(value, key) {
+					return _.toLower(key);
+				});
```

Comment:
This mapping could be made more declarative, as we only want to operate on the keys:

```suggestion
				months = _
					.chain(self.i18n.active().strategy.holidays.months)
					.keys()
					.map(_.toLower)
					.value()
```